### PR TITLE
stable/sonatype-nexus

### PR DESCRIPTION
**Pull request by cmoroianu1**
Signed-off-by: Catalin Moroianu <c.moroianu@travelaudience.com>

#### What this PR does / why we need it:
Update Sonatype Nexus to the latest version.

Changes:
- updated Nexus to latest version
- updated Alpine to latest version
- updated OpenJDK to latest version 
- update nexus-proxy image to the latest version
- update nexus-backup  image to the latest version

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sonatype-nexus/values.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ deploymentStrategy: {}
 
 nexus:
   imageName: quay.io/travelaudience/docker-nexus
-  imageTag: 3.15.2
+  imageTag: 3.17.0
```
Comment (davidkarlsen) [inline stable/sonatype-nexus/values.yaml:13]: please update the readme too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/sonatype-nexus/values.yaml (hunk 1 above)
Comment (cmoroianu1) [reply stable/sonatype-nexus/values.yaml:13]: updated. thanks!

**Comment by davidkarlsen** [comment]
@cmoroianu1 In the mean time 3.18.0 has been released. Update?

**Review by davidkarlsen — APPROVED** [review]

**Comment by davidkarlsen** [comment]
Why not use the standard image https://hub.docker.com/r/sonatype/nexus3/ ?

**Pull request merged**